### PR TITLE
Dec blog post

### PR DESCRIPTION
**Pull request by kkraune**
... or @frodelu

### CODE REVIEW

File: _posts/newsletter/2021-12-22-vespa-newsletter-december-2021.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+#### Vespa IntelliJ plugin
+[Shahar Ariel](https://github.com/shahariel) has created an IntelliJ plugin for editing schema files,
+find it at [docs.vespa.ai/en/schemas.html#intellij-plugin](https://docs.vespa.ai/en/schemas.html#intellij-plugin).
+Thanks a lot for the contribution!
+
+
+___
+About Vespa: Largely developed by Yahoo engineers,
```
Comment (bratseth) [inline _posts/newsletter/2021-12-22-vespa-newsletter-december-2021.md:64]: Didn't we agree to remove this?

Comment (kkraune) [reply _posts/newsletter/2021-12-22-vespa-newsletter-december-2021.md:64]: don't remember :-) but happy to remove - this line only, or the full section?

Comment (bratseth) [reply _posts/newsletter/2021-12-22-vespa-newsletter-december-2021.md:64]: The full section.

Comment (kkraune) [reply _posts/newsletter/2021-12-22-vespa-newsletter-december-2021.md:64]: all right :-)

**Comment by kkraune** [comment]
@bratseth PTAL

**Review by bratseth — APPROVED** [review]
👍 👍

**Pull request merged**